src/game/rdr/Scripts.hpp

```diff
@@ -0,0 +1,13 @@
+#pragma once
+#include "util/Joaat.hpp"
+
+namespace rage
+{
+	class scrThread;
+}
+
+namespace YimMenu::Scripts
+{
+	rage::scrThread* FindScriptThread(joaat_t hash);
+	void RunAsScript(rage::scrThread* thread, std::function<void()> callback);

```

Comment:
Might need to be declared as `extern` if you're going to include `Scripts.hpp` into multiple different files.